.circleci/config.yml

```diff
@@ -188,6 +205,13 @@ workflows:
           filters:
             branches:
               only: master
+      - release-dry-run:
+          context: Tilt Slack Context
+          requires:
+            - build-linux
+          filters:
+            branches:
+              only: master
```

Comment:
Seem ok to run this on master? Or should we go back to the trigger on demand like the previous dry run workflow?